package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "talkly",
+  "scripts": {
+    "postinstall": "./node_modules/.bin/bower install && ./node_modules/.bin/gulp",
+    "start": "./node_modules/.bin/gulp watch"
+  },
+  "dependencies": {
+    "bower": "^1.7.9",
+    "gulp": "^3.9.0",
+    "gulp-concat": "^2.6.0",
+    "gulp-minify-css": "^1.2.1",
```

Comment:
In this case, yes we have only sass here. But if you combine it with vendor css files then you need to compress them in diferent way. But yes, I fix this.

Btw. We are thinking about switching to reactjs with webpack.